models/preles/R/write.config.PRELES.R

```diff
@@ -7,6 +7,23 @@
 # http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
 
+add.samples.PRELES <- function(trait.samples){
```

Comment:
it's not clear to me what this function does. It appears to just rename PRELES variables, but those variables names don't appear to be used in runPRELES.jobsh